test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -925,7 +926,9 @@ def revert_mom(self, mom, force=False):
         name = mom.shortname
         if mom.platform == 'cray' or mom.platform == 'craysim':
             try:
-                self.server.manager(MGR_CMD_DELETE, NODE, None, '')
+                if self.del_all_nodes:
```

Comment:
I still think @hirenvadalia has a good point.  A comment in the code stating the same thing you said in your response would be useful.